icu4c/source/common/uloc.cpp

```diff
@@ -598,7 +592,7 @@ compareKeywordStructs(const void * /*context*/, const void *left, const void *ri
     return uprv_strcmp(leftString, rightString);
 }
 
-static void
+void U_EXPORT2
 _getKeywords(const char *localeID,
```

Comment:
For a function that is no longer file-local, please also give it a uloc_ prefix.

Reply:
Done.